Bandwidth.Standard/WebRtc/Utils/WebRtcTransfer.cs

```diff
@@ -1,17 +1,34 @@
-using System.ComponentModel;
-
 namespace Bandwidth.Standard.WebRtc.Utils
 {
     public static class WebRtcTransfer {
-        public static string generateBxml(string deviceToken) {
-            return WebRtcTransfer.generateBxml(deviceToken, "sip:sipx.webrtc.bandwidth.com:5060");
+        private const string SipUri = "sip:sipx.webrtc.bandwidth.com:5060";
+
+        public static string GenerateBxml(string deviceToken, string sipUri = SipUri)
+        {
+            var bxmlVerb = GenerateBxmlVerb(deviceToken, sipUri);
+            return GenerateBxml(bxmlVerb);
+        }
+        
+        public static string GenerateBxmlVoiceCallId(string deviceToken, string voiceCallId, string sipUri = SipUri)
```

Comment:
Is it a requirement to use a different method name? Can methods be overloaded in C#?

Reply:
I'll comment here since mine is related - is it necessary to have both methods if we are making this a breaking change? Could we just have `GenerateBxmlVerb(string deviceToken, string voiceCallId, string sipUri = SipUri)`?

Reply:
I agree with Audrey. That also keeps it consistent with what we've done for other languages.

Reply:
Methods can be overloaded in C# but I thought it was more clear to have the distinction between them since the `voiceCallId` and `sipUri` are being treated differently. I moved the parameters around a bit though and am happy the result (and not having two methods).